_posts/2020-03-31-Ordina-Young-Professional-Programme-2020-Light-Edition.md

```diff
@@ -0,0 +1,206 @@
+In the first part of this session we took a dive into the basics of HTML and CSS, 
+and after this we learned about the more recent features of HTML5 and CSS3. 
+The remainder of the session was mainly about JavaScript. 
+Throughout the session we had some exercises to have a bit of a hands-on approach.
+    
+
+
+### Typescript
+<img class="image left medium" alt="Typescript logo" src="/img/2020-03-31-Young-Professional-2020-Light/typescript.png">
+
+Building on the basics we learned prior, we had a session about Typescript, 
+a derivative of Javascript that is becoming more and more popular amongst most front end frameworks 
+because of it's similarity to Javascript but also having types allowing stricter rules to be enforced 
+and less random unexplainable errors. 
+Similarly to the previous session we got an explanation on the basics of Typescript which was then used in some exercises.
+
+
+### Angular
+<img class="image right medium" alt="Angular logo" src="/img/2020-03-31-Young-Professional-2020-Light/angular.png">
+
+Eventually we reached the end of our front end training with a 2-day Angular course. 
+Everything we learnt the previous days was now being poured into a framework. 
+In this course we, again, saw the basics of this massive Typescript framework. 
+We saw everything from dependency injection to property binding. 
+Angular is a framework built by Google so there are lots of features, 
+because of this we didn't have enough time in those 2 days to really explore Angular in-depth. 
+This is the preferred front end technology in JWorks. 
+We were also going to use it in the dev case, so we would have enough time to understand it on a deeper level.
+
+
+# Cloud
+
+### Docker
+<img class="image left medium" alt="Docker logo" src="/img/2020-03-31-Young-Professional-2020-Light/docker.png">
+
+To start off our cloud education we had a session about Docker, which is essential to many cloud technologies. 
+Tom Verelst gave an interesting hand-on session where he first explained the concepts of Docker, 
+then we started writing our own yaml deployment files to set up our first full stack application.
+
+### Kubernetes
+<img class="image right medium" alt="Kubernetes logo" src="/img/2020-03-31-Young-Professional-2020-Light/kubernetes.png">
+
+Kubernetes is an open source container orchestration framework, which is commonly used to deploy applications. 
+Pieter Vincken taught us the basics of Kubernetes. 
+We started with a presentation about the core concepts like pods, secrets and more. 
+Then we learned how to deploy and manage our Docker containers using Kubernetes. 
+During this session we got the hang of how you could manage your clusters by using various techniques provided by Kubernetes 
+like auto scaling, load balancing,... 
+It was really insightful to see how easy it is to deploy and manage your applications on a Kubernetes cluster. 
+
+### Security Principles
+In this day and age security is a hot topic. 
```

Comment:
Let's also add an image here?
You could use the lock image from Jeroen Meys' weekly security quiz.
Let me know if you need help with trimming its canvas.